src/main/java/sirius/biz/translations/mongo/MongoTranslations.java

```diff
@@ -29,6 +28,15 @@ public MongoTranslations(BaseEntity<?> owner) {
         super(owner);
     }
 
+    @Override
+    protected void deleteAllTranslations() {
+        for (MongoTranslation t : mango.select(MongoTranslation.class)
```

Comment:
It's possible to delete all in a single command based on filter instead of looping and deleting one by one.